source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/inputhandler/validation/TextInputRangeValidator.java

```diff
@@ -0,0 +1,33 @@
+package io.adaptivecards.renderer.inputhandler.validation;
+
+import android.widget.EditText;
+
+import java.util.regex.Pattern;
+
+import io.adaptivecards.objectmodel.TextInput;
+
+public abstract class TextInputRangeValidator extends ActualValidator
```

Comment:
just curious, are we going to have Range/Regex/Required Validators for other type of input such as Number?